<launch>
    <arg name="output" default="screen"/>
    <arg name="calib" default="false"/>
    <arg name="camera" default="flir_gige"/>

    <node pkg="rosbag" type="play" name="player" output="$(arg output)"
        args="-l /home/chao/Workspace/bag/flir/flir_gige_2.bag">
    </node>

    <node pkg="flir_gige" type="calib_proc_node" name="calib_proc"
        ns="$(arg camera)" output="screen">
        <param name="thresh_window" type="int" value="17"/>
        <param name="thresh_type" type="int" value="0"/>
    </node>

    <node pkg="image_view" type="image_view" name="viewr"
        args="image:=/$(arg camera)/image_calib">
    </node>

    <node if="$(arg calib)" pkg="camera_calibration" type="cameracalibrator.py"
        name="calibrator"
        args="camera:=/$(arg camera) image:=/$(arg camera)/image_calib
        --pattern circles --size 5x4 --square 0.028 --no-service-check">
    </node>

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="reconfigure">
    </node>
</launch>
